tests/bats/validate/govcms-validate-profile.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+# shellcheck disable=SC2002,SC2031,SC2030,SC2155
+
+load ../_helpers_govcms
+
+@test "Check profile: defaults" {
+  run scripts/validate/govcms-validate-profile >&3
+  echo "GovCMS Validate :: Validate install profile"
```

Comment:
@steveworley just noticed that now while working on another test; was this meant to be an `assert_output_contains` instead of `echo`?

Reply:
```suggestion
  assert_output_contains "GovCMS Validate :: Validate install profile"
```